src/databricks/labs/remorph/helpers/reconcile_config_utils.py

```diff
@@ -0,0 +1,390 @@
+import json
+import logging
+import webbrowser
+
+from databricks.connect import DatabricksSession
+from databricks.labs.blueprint.installation import Installation, SerdeError
+from databricks.labs.blueprint.tui import Prompts
+from databricks.labs.remorph.config import TableRecon, ReconcileConfig, get_dialect
+from databricks.labs.remorph.reconcile.connectors.source_adapter import create_adapter
+from databricks.labs.remorph.reconcile.constants import SourceType
+from databricks.sdk import WorkspaceClient
+from databricks.sdk.errors.platform import ResourceDoesNotExist, NotFound, PermissionDenied
+
+logger = logging.getLogger(__name__)
+
+README_RECON_CONFIG_NOTEBOOK = """# Databricks notebook source
+# MAGIC %md
+# MAGIC # Recon Config setup instructions (see [README](readme_link))
```

Comment:
don't you want to replace `readme_link` with a real link?

Reply:
this is just a template for now, we'll add more detailed examples and links